bin/docker-setup

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
Note that the Ruby version could have been invoked from anywhere via  a relative path and still worked, thanks to the 

```ruby
FileUtils.chdir APP_ROOT do
  ...
end
```

block.

While I don't think that's critically important, the bash version doesn't offer this,  so if you feel like it, maybe you want to add it.

Reply:
@citizen428 I agree with you that it isn't critically important since most of us would just run the command on the working directory as specified on the docs. But if there's a need for it, one can simply add that later on.

Reply:
Agreed, let's keep as is.